packages/common.vm/tools/vm.common/vm.common.psm1

```diff
@@ -621,7 +623,20 @@ function VM-Install-With-Installer {
         Install-ChocolateyInstallPackage @packageArgs
         VM-Assert-Path $executablePath
 
-        VM-Install-Shortcut -toolName $toolName -category $category -executablePath $executablePath -consoleApp $consoleApp -arguments $arguments
+        # Set the default icon to be the executable's icon. For MSI files, attempt to get executable icon from installer first.
+        if ($fileType -eq 'MSI') {
+            $iconPath = VM-Get-MSIInstallerPathByProductName $toolName
```

Comment:
I like the feature to provide tools installed from an MSI an icon, but the code is a bit complicated, so thinking of ways to simplify it:
- can we get the icon from the installed `$executablePath`? instead of looking for the icon in the files?

Any other ideas?

Reply:
I would love a simpler way if possible!

I could not find the icon via `$executablePath` for this case because the icon is not associated with anything that is in the installed folders, surprisingly, which is what led me down this path.